src/models/Product.js

```diff
@@ -3,59 +3,61 @@ const { PRODUCT_SIZES } = require('../constant')
 
 // TODO: create reviews model
 const ProductSchema = new mongoose.Schema(
-  {
-    name: {
-      type: String,
-      required: true,
-    },
-    images: [
-      {
-        src: { type: String, required: true },
-        color: { type: String, required: false },
-        description: { type: String, required: false },
-      },
-    ],
-    price: {
-      type: Number,
-      required: true,
-    },
-    sizes: [
-      {
-        stock: {
-          type: Number,
-          default: 0,
-          validate(value) {
-            if (value < 0) {
-              throw new Error("stock can't be less than 0")
-            }
-          },
+    {
+        name: {
+            type: String,
+            required: true,
         },
-        label: {
-          type: String,
-          required: true,
-          enum: PRODUCT_SIZES,
+        images: [
+            {
+                src: { type: String, required: true },
+                color: { type: String, required: false },
+                description: { type: String, required: false },
+            },
+        ],
+        price: {
+            type: Number,
+            required: true,
+        },
+        sizes: [
+            {
+                stock: {
+                    type: Number,
+                    default: 0,
+                    validate(value) {
+                        if (value < 0) {
+                            throw new Error("stock can't be less than 0")
+                        }
+                    },
+                },
+                label: {
+                    type: String,
+                    required: false,
+                    enum: PRODUCT_SIZES,
+                },
+            },
+        ],
+        description: {
+            type: String,
+            required: false,
+        },
+        discount: {
+            type: Number,
+            default: 0,
+            validate(value) {
+                if (value < 0 || value > 100) {
+                    throw new Error(
+                        "disscount can't be less than 0 or greater than 100"
+                    )
+                }
+            },
+        },
+        category: {
+            type: mongoose.Schema.Types.ObjectId,
+            ref: 'Category',
         },
-      },
-    ],
-    description: {
-      type: String,
-      required: false,
-    },
-    discount: {
-      type: Number,
-      default: 0,
-      validate(value) {
-        if (value < 0 || value > 100) {
-          throw new Error("disscount can't be less than 0 or greater than 100")
-        }
-      },
-    },
-    category: {
-      type: mongoose.Schema.Types.ObjectId,
-      ref: 'Category',
     },
-  },
-  { timestamps: true }
+    { timestamps: true }
```

Comment:
i added the timestamps that u deleted by mistake for some reason xD

Reply:
fuck you